### PR TITLE
Add Backpack to betafpv 1W, and device_name

### PR DESCRIPTION
**Pull request by CapnBry**
The BetaFPV 1000mW target has a backpack but it is not included in the target definition.

* Added USE_TX_BACKPACK
* Added DEVICE_NAME "BFPV 2G4Micro1W" (15 chars)
* Shorten DEVICE_NAME "BETAFPV 2G4Micro" to "BETAFPV2G4Micro" (16 to 15 chars)

### CODE REVIEW

**Comment by maybenikhil** [comment]
Is this not equivalent to #1275

**Comment by pkendall64** [comment]
We already have a different target for the 1W so fixing it to include the backpack is the right move.
We should probably just close the other PR? I know some of us have samples now.

**Review by pkendall64 — APPROVED** [review]

**Comment by CapnBry** [comment]
> Is this not equivalent to #1275

Oh uhhhh I guess so, but there are two targets. Does the original recipe version have the backpack as well?

**Comment by maybenikhil** [comment]
It does not! closing #1275

**Review by maybenikhil — APPROVED** [review]

**Comment by pkendall64** [comment]
Actually, I think Alessandro said it did have the backpack.

**Comment by maybenikhil** [comment]
😅

File: src/include/target/BETAFPV_2400_TX_MICRO.h [hunk 1]
```diff
@@ -37,6 +41,8 @@
     #define MinPower PWR_25mW
     #define MaxPower PWR_1000mW
     #define POWER_OUTPUT_VALUES {-18,-15,-12,-7,-4,2}
+    // Backpack on standard pins 1/3
+    #define USE_TX_BACKPACK
```
Comment (pkendall64) [inline src/include/target/BETAFPV_2400_TX_MICRO.h:45]: Move to the top with other features as it's on both types

**Comment by JyeSmith** [comment]
Full duplex needs to be confirmed before merging.

**Comment by pkendall64** [comment]
> Full duplex needs to be confirmed before merging.

What full duplex?

**Comment by JyeSmith** [comment]
Full duplex communication between the esp32 and 8285 needs to be confirmed working.

**Comment by pkendall64** [comment]
> Full duplex communication between the esp32 and 8285 needs to be confirmed working.

Confirmed by @AlessandroAU and myself on discord.

**Comment by CapnBry** [comment]
And @me too! Backpack works on my sample unit after pkendall64's PR.

**Pull request merged**